<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FLOUWER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>461</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="button_quedarme_embarazada_2">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Quedarme 
Embarazada</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_entendermicuerpo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Entender mi cuerpo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button_gestionar_ciclo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Gestionar 
 elciclo mestrual</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_nada_2">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Nada de lo anterior </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecciona una opcion </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>90</height>
     </size>
    </property>
    <property name="text">
     <string>Iniciar Sesion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>En que podemos ayudarte</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bienvenida</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
